.travis.settings.xml

```diff
@@ -6,4 +6,15 @@
 			<password>${env.OSSRH_PASSWORD}</password>
 		</server>
 	</servers>
+	<profiles>
```

Comment:
Isn't it a bit confusing to use two profiles (one here and one in the `pom.xml`)? GPG is only needed during deployment, so `activateByDefault` is also a bit misleading. What do you think?